### ISSUE TITLE
[Feature Request] "Join Builder"

### ISSUE THREAD

**Issue by MrMage** [TablePlus/TablePlus#2084]
I often need to join two tables based on fairly obvious IDs (e.g. joining books with authors on `book.author_id = author.id`). Having to type the join syntax over and over again is cumbersome. It would be great if TablePlus could let me automatically join two related tables, e.g. from the context menu of one of the tables.

If this is already possible, please let me know.

**Comment by glaucocustodio** [comment]
I agree, this would be a big win in productivity, https://www.pgcli.com/ has it and I find super useful.

**Comment by huyphams** [comment]
Good news, we're testing it

<img width="769" alt="Screen Shot 2022-05-05 at 8 02 28 PM" src="https://user-images.githubusercontent.com/3654622/166928430-b0829480-2d05-414a-bb0b-7958d39bfb69.png">

**Comment by glaucocustodio** [comment]
oh my gosh, thank you so much @huyphams, I am available to be a beta tester if you need

**Comment by huyphams** [comment]
Hi @glaucocustodio  you can download the patch here: https://www.dropbox.com/s/kjeon87wr82g2ps/TablePlus.dmg?dl=0 
feel free to give us any feedback.

**Comment by glaucocustodio** [comment]
It looks really great, but I noted it only works for columns that have foreign key. If somehow it could be smart enough to identify relationships lacking foreign key would be perfect, but I believe it can come next.

**Comment by huyphams** [comment]
Yeah, this is the first version. We will try to improve it in future updates. We do have a plan to rebuild the Abstract syntax tree (AST) to make the auto-completion better.
